ServerModules.cfg

```diff
@@ -29,6 +29,7 @@ HDF5MultiDim
 ParquetMsg
 EncodingMsg
 GroupByMsg
+BigInt
```

Comment:
Just noting that this isn't really an "optional" module (e.g., commenting out this will still compile the code since the module is used in a couple of modules that are always built into the server), so it might not make sense to put in the `ServerModules.cfg` file.

Reply:
(FWIW, I might remove the `BigInt` module from this list and then just add a `use BigInt;` in the `ServerDaemon.chpl` file, like we do with other non-optional modules)

Reply:
Sounds good!

Reply:
done!